config/config.toml.example

```diff
@@ -283,6 +283,10 @@ batch-wait-size = 8
 # Enable chunk encoded data.
 enable-arrow = true
 
+# If a Region has not been accessed for more than the given duration, it
```

Comment:
Do we need to add the unit of `region-cache-ttl` to the comment (second)?

Reply:
Good idea 👍